package.json

```diff
@@ -1,47 +1,21 @@
 {
-  "name": "create-pkg",
-  "version": "0.0.0",
-  "description": "Scaffold a package",
-  "author": "Wes Todd <wes@wesleytodd.com>",
-  "keywords": [
-    "scaffold",
-    "package",
-    "git",
-    "module",
-    "package.json",
-    "readme",
-    "mocha",
-    "standard"
-  ],
-  "license": "ISC",
-  "main": "index.js",
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:pkgjs/create.git"
-  },
-  "bin": {
-    "create-package": "bin/create-package"
-  },
+  "name": "create-pkg-monorepo",
```

Comment:
On Naming: this name doesn't really do anything, but I think we should choose a name for the repo and make this the same. On one of the calls we discussed moving this repo back to the name `create`, if we do that we should just make this `create` as well (or at most `@pkgjs/create`).

Additionally, we should name the `packages/*` directories the same name as the package. So that means `packages/create-pkg` (drop the cli suffix). That makes it much easier for folks to find what they are looking for by name.